services/web/server/src/simcore_service_webserver/projects/_projects_service.py

```diff
@@ -275,6 +275,17 @@ async def batch_get_project_name(
     return [name if name else "Unknown" for name in get_project_names]
 
 
+async def batch_get_projects(
+    app: web.Application,
+    *,
+    project_uuids: list[ProjectID],
+) -> list[ProjectDBGet]:
```

Comment:
I think the idea here is that ProjectDB should not leave to the outside world. maybe I am mistaken @pcrespov ?

Reply:
WE have to align here... I have a different understanding of @matusdrobuliak66 concept for `*DB` models
@giancarloromeo perhaps we can refactor together `proejcts` and make sure we are in the same page